src/jit-format/jit-format.cs

```diff
@@ -0,0 +1,682 @@
+                        _rootPath = (found) ? rootPath : _rootPath;
+
+                        // Set up compileCommands
+                        var compileCommands = ExtractDefault<string>("compile-commands", out found);
+                        _compileCommands = (found) ? compileCommands : _compileCommands;
+
+                        // Set flag from default for verbose.
+                        var verbose = ExtractDefault<bool>("verbose", out found);
+                        _verbose = (found) ? verbose : _verbose;
+
+                        // Set up untidy
+                        var untidy = ExtractDefault<bool>("untidy", out found);
+                        _untidy = (found) ? untidy : _untidy;
+
+                        // Set up noformat
+                        var noformat = ExtractDefault<bool>("noformat", out found);
+                        _noformat = (found) ? noformat : _noformat;
+
+                        // Set up fix
+                        var fix = ExtractDefault<bool>("fix", out found);
+                        _fix = (found) ? fix : _fix;
+                    }
+                }
+                else
+                {
+                    Console.Error.WriteLine("Can't find format.json on {0}", _jitUtilsRoot);
+                }
+            }
+            else
+            {
+                Console.WriteLine("Environment variable JIT_FORMAT_ROOT not found - no configuration loaded.");
+            }
+        }
+
+        public static CommandResult TryCommand (string name, IEnumerable<string> commandArgs, bool capture = false)
+        {
+            try 
+            {
+                Command command =  Command.Create(name, commandArgs);
+
+                if (capture)
+                {
+                    // Capture stdout/stderr for consumption within tool.
+                    command.CaptureStdOut();
+                    command.CaptureStdErr();
+                }
+                else
+                {
+                    // Wireup stdout/stderr so we can see output.
+                    command.ForwardStdOut();
+                    command.ForwardStdErr();
+                }
+
+                return command.Execute();
+            }
+            catch (CommandUnknownException e)
+            {
+                Console.Error.WriteLine("\nError: {0} command not found!  Add {0} to the path.", name, e);
+                Environment.Exit(-1);
+                return CommandResult.Empty;
+            }
+        }
+
+        public T ExtractDefault<T>(string name, out bool found)
+        {
+            var token = _jObj["default"][name];
+
+            if (token != null)
+            {
+                found = true;
+
+                try
+                {
+                    return token.Value<T>();
+                }
+                catch (System.FormatException e)
+                {
+                    Console.Error.WriteLine("Bad format for default {0}.  See asmdiff.json", name, e);
```

Comment:
Error line still notes asmdiff.json.  Should be reconciled with which ever way you go with the config file.